fourkitchens/sous-drupal-recipe
# Sous media recipe

### Description
Installs media library and creates three media types with the same fields, form and view display, and image styles that we had in the original Sous distro.

- file
- image
- video (remote YouTube and Vimeo included)

### Included the following contrib modules:

1. focal_point (also included in original distro)
2. media_library_edit (a tiny module that adds an edit button to the media library so that it is possible to edit media in the content edit dialogue)

### Testing steps:

1. Run through the install process included in the README. You should see this recipe applied after sous_base and sous_admin.
2. Use the provided link to log in. Verify the media types and image styles are present.
